### DISCUSSION TITLE
select 能根据option的value来选择吗？

### DISCUSSION THREAD

**Discussion by ahking** [layui/layui#2039] [问答]
<select lay-search=""> 如果option包括输入的某几个字母就列出对应选项

**Comment by Sight-wcg** [comment]
> select 能根据option的value来选择吗？

select 只能根据 option 列表中选取的 option 来选择，内部也是根据 option 元素的 value 属性值来选择默认选中项的

> 如果option包括输入的某几个字母就列出对应选项

是的， select 搜索时的行为是这样的
